src/catalogue/app/src/main/java/catalogue/controllers/CatalogueItemDTO.java

```diff
@@ -47,76 +49,87 @@ public CatalogueItemDTO(
 
     /**
      * The id of the product
+     * @return 
```

Comment:
Why these empty return tags?

Reply:
I've added it because there was "Missing @return tag" warning and it was added as quick fix.

Reply:
ok